### ISSUE TITLE
Remove UV-index units

### ISSUE THREAD

**Issue by bhouseski** [briis/meteobridge#30]
Is there a way to remove the units for U -index?  I am used to this being unit less.

**Comment by briis** [comment]
Yes, that is possible. However I have not looked at this integration for a long time, so it will take a a little time to bring it up to 2024 HA standards.

**Comment by iu1jvo** [comment]
@bhouseski  have you try to create a Template Sensor, based on the Meteobridge sensor? 
https://www.home-assistant.io/integrations/template/
